playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+
+As with the tech planning process itself, creating decision matrices benefits from [collaboration and feedback](https://review.firstround.com/this-matrix-helps-growing-teams-make-great-decisions).
+
+## Stakeholders
+Stakeholders represent the list of people from whom you require tech plan approval. Think of this as similar to the "Reviewers" field on Github PRs. As the author of the tech plan, you are responsible for choosing appropriate stakeholders to arrive at the best technical approach.
+
+Some guidelines for choosing stakeholders:
+- Aim for ~3-5 stakeholders.
+- Stakeholders should be from the engineering team, given the tech plan is focused on implementation decisions. Of course it's great to share and collaborate outside of engineering, but other team members need not be on this list.
+- Choose stakeholders whose expertise is important for ensuring the best decision is made. If the change affects another team's domain, include someone from that team. Consider including your team's tech lead as well as relevant staff engineers.
+
+## Receiving Feedback
+Since all developers are invited to participate on a tech plan and leave feedback, setting clear expectations on the tech plan's timeline can help ensure this process is smooth. We recommend filling out the Feedback requested by field.
+
+If there is someone that you explicitly want feedback from, include them in the Stakeholders section (and don't hesitate to remind them directly, Notion notifications are easily lost).
+
+Significant tech plans benefit from a synchronous review in a practice meeting. If there is not a practice meeting that works for your timeline, schedule something ad-hoc. Make sure to mention the tech plan in our weekly engineering standup to invite broader feedback.
+
+## Giving Tech Plan Feedback
+Feedback is expected and encouraged. Similar to leaving PR comments, please distinguish between blocking and non-blocking feedback so that your intent is clear.
+
+If you have questions about or significant feedback on the overall direction of the tech plan, we recommend asking the tech plan author to hop on a quick call to discuss.
+
+## Reviewing and Approving a Tech Plan
+Stakeholders are required to formally review the tech plan (also in a similar manner as Github PRs) by marking the plan with the appropriate review status:
+- **Approved** means that you are OK with the direction proposed by the plan.
+- **Rejected** means that you are not OK with the proposed direction.
+- **Request changes** means that you would be OK with the proposed direction, provided some changes (that you clarify in the Comments section) are addressed.
+
+Based on these review statuses, the tech plan author can make changes and ask for a subsequent review.
```

Comment:
What happens if a stakeholder (or `reviewer`, depending on what we end up calling them) marks something as `Rejected`? Who gets the final say as to if the plan is implemented or not in this scenario? (Sorry if I just missed this somewhere!).

Reply:
Good question – based on the feedback above, I think I'll remove the "Rejected" status in favor of "Changes requested". I'd say the writer is still responsible for the ultimate decision, but they are expected to take the feedback from reviewers seriously and addressing the requested changes.